wasmcloud_host/lib/wasmcloud_host_web/controllers/linkdefs_controller.ex

```diff
@@ -0,0 +1,30 @@
+defmodule WasmcloudHostWeb.LinkdefsController do
+  use WasmcloudHostWeb, :controller
+  require HostCore
+
+  def define_link(conn, params) do
+    actor = params["actor_id"]
+    contract_id = params["contract_id"]
+    link_name = params["link_name"]
+    provider_key = params["provider_id"]
+    values =
+      params["values"]
+      |> String.split(",")
+      |> Enum.flat_map(fn s -> String.split(s, "=") end)
+      |> Enum.chunk_every(2)
+      |> Enum.map(fn [a, b] -> {a, b} end)
+      |> Map.new()
+
+    IO.inspect(values)
+
+    HostCore.LinkdefsManager.put_link_definition(
```

Comment:
I love that this will also advertise said linkdef throughout the lattice, to all other running hosts... and therefore make all other washboard consoles update :)